package.json

```diff
@@ -5,6 +5,43 @@
   "main": "dist/index.js",
   "module": "dist/va-forms-system-core.cjs.production.min.js",
   "typings": "dist/index.d.ts",
+  "release": {
+    "branches": [
+      "main"
+    ],
+    "plugins": [
+      [
+        "@semantic-release/commit-analyzer",
+        {
+          "preset": "angular",
+          "releaseRules": [
+            {
+              "type": "fix",
+              "release": "patch"
+            },
+            {
+              "type": "feat",
+              "release": "minor"
+            },
+            {
+              "type": "perf",
+              "release": "major"
+            }
+          ],
+          "parserOpts": {
+            "noteKeywords": [
+              "BREAKING CHANGE",
+              "BREAKING CHANGES"
+            ]
+          }
+        }
+      ],
```

Comment:
This is pretty cool. Will this require all of our commits (or at least 1 commit per PR) to use this angular convention?

Reply:
Exactly so it goes through the commits and analyzes the messages, looking for (fix, feat, or perf "With perf you have to use the BREAKING CHANGES: at the footer of the commit message).